MobileNav/ViewControllers/Map/SearchViewController.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  SearchViewController.swift
+//  MobileNav
+//
+//  Created by James Lapinski on 5/13/20.
+//  Copyright © 2020 Livio Inc. All rights reserved.
+//
+
+import MapKit
+import UIKit
+
+protocol SearchViewControllerDelegate: class {
+    func didSelectPlace(coordinate:CLLocationCoordinate2D)
+}
+
+class SearchViewController: UIViewController {
+    @IBOutlet weak var searchBar: UISearchBar!
```

Comment:
Can you set some placeholder text in the search bar so it is easier to understand what you are supposed to search for: something like "Search for restaurants, coffee shops, and hardware stores..."